### DISCUSSION TITLE
Hacktribe Booster Kit

### DISCUSSION THREAD

**Discussion by richardarnezjames** [bangcorrupt/hacktribe#250] [General]
I've collected a near complete history lesson of Korg drum machine samples for the Hacktribe community. All at 48k 24bit for ease of transfer. @bangcorrupt how do I add that to the Content tree?

**Comment by enorrmann** [comment]
I'm interested! do you have a download link ?

**Comment by BKLronin** [comment]
Really awesome, seen this in the timeline. Got the Triton samples already installed but getting import error with OeSampleeditor. 

- I think the samples for import are 16 bit 48 kHz. The DAC is 24 bit though. At least my editor is coverting them to 16 bit
- Noticed some errored files with endings in EMX folder.
- Mono would be cool, conversion doesnt always work. Triton good, EMX gets stuck.

Doubt itś added to the content because of legal reasons but it is a great collection anyways.

**Comment by richardarnezjames** [comment]
yea i thought about that, i'll upload them to my google one and post the link, I'll include a mono conversion of them all and convert the stereo ones to 16 bit

**Reply by BKLronin** [reply]
Just a suggestion bangcorrputs decision after all  :)

I have no idea why the triton samples make zero problems and the EMX ones do. I really missed them since I sold it many years ago. I suspect some filename is problematic as I tried another samplepack from someone else and it fails too with the editor. 

Gonna experiment a bit but I think mono 16 is less complicated. Maybe seperate folder so the originals are conserved?

The import errors because of the stereo. The 25 to 16 bit conversion seems good so not file related at all.

**Comment by bangcorrupt** [comment]
Thank you for the samples, but I don't think I can add them to Hacktribe.  If they are exported directly from the machines they are usually covered by copyright.  

> samples for import are 16 bit 48 kHz. The DAC is 24 bit 

The DSP processing is also 24 bit.  Samples imported individually from the SD card are converted from 24 bit to 16 bit.

> problems

Sometimes unexpected metadata sections can cause issues.  Converting with something simple like sox or audacity can often fix it, even without changing the format.

**Reply by richardarnezjames** [reply]
The electribe, wave____, m1 and DSS and volcas samples were not exported. I cant speak on the rest of them tho. I'll try that thanks. I'll have a link ready for anyone who wants the full library and upload just the ones I sampled.

**Reply by bangcorrupt** [reply]
If they are raw samples of the factory sounds I think they are still copyright.  If they are new sounds sculpted from the source material then it is definitely ok.  If they make it through [freesound](https://freesound.org) moderation with a clear description of their source and a CC0 license then I'll definitely add a selection to the content repo.

**Reply by richardarnezjames** [reply]
Okay good to know I'll try that

**Comment by djokenzi** [comment]
@richardarnezjames Do you have a download link for those samples? Thanks in advance!

**Comment by richardarnezjames** [comment]
Mono files as requested.

https://drive.google.com/drive/folders/1zKTugyEUGAn_4RzIV0N1Xl_yNp5fo9fc?usp=share_link
